packages/react/package.json

```diff
@@ -65,6 +65,7 @@
     "css-loader": "^6.7.4",
     "identity-obj-proxy": "^3.0.0",
     "jest-axe": "^6.0.1",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
@joshfarrant so I know, is this related to storybook upgrade, or a missing dependency in the project? I assumed jsdom environment was already set.

Reply:
Yes, it's related to the Storybook upgrade.

Storybook 8 requires Jest 28, and in Jest 28 they [stopped bundling `jest-environment-jsdom` as part of Jest](https://jestjs.io/blog/2022/04/25/jest-28#breaking-changes).

> Please note that both of the removed modules (jest-environment-jsdom and jest-jasmine2) are still actively maintained and tested in the same way, so the only breaking change here is that you'll need to explicitly install them.